app/src/main/java/de/westnordost/streetcomplete/quests/defibrillator/AddDefibrillatorLocationForm.kt

```diff
@@ -0,0 +1,28 @@
+package de.westnordost.streetcomplete.quests.defibrillator
+
+import android.os.Bundle
+import android.view.View
+import androidx.core.widget.doAfterTextChanged
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.databinding.QuestLocationBinding
+import de.westnordost.streetcomplete.quests.AbstractOsmQuestForm
+import de.westnordost.streetcomplete.util.ktx.nonBlankTextOrNull
+
+class AddDefibrillatorLocationForm : AbstractOsmQuestForm<DefibrillatorLocation>() {
+
+    override val contentLayoutResId = R.layout.quest_location
+    private val binding by contentViewBinding(QuestLocationBinding::bind)
+
+    private val location get() = binding.locationInput.nonBlankTextOrNull
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.locationInput.doAfterTextChanged { checkIsFormComplete() }
+    }
+
+    override fun onClickOk() {
+        applyAnswer(DefibrillatorLocation(location!!))
+    }
+
+    override fun isFormComplete() = location != null && location!!.length >= 10
```

Comment:
Minimum description length should be discussed.

Reply:
And maximum length. Allowed in OSM are only 255 unicode character points

Reply:
Maximum length is limit by android:maxLength="253"